README.md

```diff
@@ -76,7 +76,11 @@ Alternatively,
 Finally, tag and push the new release
 
 * Patch, Minor or Major Version the tag according to `<Major>.<Minor>.<Patch>`
-* `git push --tag`
+* `git push origin <Major>.<Minor>.<Patch>` to push your newly created tag to Github.
+
+> [!WARNING]
+> Avoid using `git push --tags` as it will push all of your local tags to the remote repository.  
+> This is not recommended as it can cause confusion and potential issues with the build server when it attempts to calculate the release version number due to the potential of unexpected tags been pushed.
```

Comment:
```suggestion
> This is not recommended as it can cause confusion and potential issues with the build server when it attempts to calculate the release version number due to the potential of unexpected tags being pushed.
```